samila/genimage.py

```diff
@@ -38,7 +44,10 @@ def __init__(self, function1=None, function2=None, data=None, config=None):
         :type data: (io.IOBase & file)
         :param config: generative image config
         :type config: (io.IOBase & file)
+        :param func_seed: random seed for function generation
+        :type func_seed: Any
         """
+        random.seed(func_seed)
```

Comment:
```python random.seed(None)``` is equal to `No seed` mode.
But, I suggest that the function should be called only when it is not None.
```python
if func_seed is not None:
    random.seed(func_seed)
```